sidebar.c

```diff
@@ -195,6 +196,16 @@ static const char *sidebar_format_str(char *buf, size_t buflen, size_t col, int
         optional = false;
       break;
 
+    case 'o':
+      if (!optional)
+      {
+        snprintf(fmt, sizeof(fmt), "%%%sd", prec);
+        snprintf(buf, buflen, fmt, m->msg_unread - m->msg_new );
```

Comment:
You'll probably want to use `msg_count` instead of `msg_unread` since `msg_new` does not include old, unread emails.

Reply:
I understand the concept of old messages and as mark_old works:
`set mark_old=no`
NEW = UNREAD

`set mark_old = yes`
NEW = unseen UNREAD 
OLD = already seen UNREAD

so `unread – new` seems right to me.

Reply:
Sorry. I wrote the explanation wrong. NEW = unseen UNREAD + already seen UNREAD.

the behaviour is consistent with `status_format ` expando `%o`. The code is almost the same as  mentioned status_format https://github.com/neomutt/neomutt/blob/master/status.c#L240

Reply:
… or simly NEW = UNREAD no matter what is `mark_old` set to.